man/systemctl.xml

```diff
@@ -1502,6 +1502,17 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-P</option></term>
+
+        <listitem>
+          <para>This is similar to <option>-p</option>/<option>--property=</option> above, but skips the
+          property name <literal>=</literal>, as if <option>--value</option> was used. Note that using
+          <option>-P</option> once will also affect all proprties listed with
+          <option>-p</option>/<option>--property=</option>.</para>
```

Comment:
I'd just really say "-P" is equivalen to "--value --property=" I must say. Says the same thing and is shorter. But it's fine this way too.

Reply:
I wanted to avoid repeating the same text as in --help...